app/design/frontend/Elogic/maksym/Magento_Checkout/templates/cart/minicart.phtml

```diff
@@ -0,0 +1,65 @@
+<li data-block="minicart" class="minicart-wrapper">
```

Comment:
Can you please add annotations for all overridden templates? Something like:
```
/**
 * Magento Checkout - Minicart
 * Overridden to add svg icon and modal
 */
```

Reply:
done